SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -275,6 +276,38 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	}
 }
 
+void USpatialNetDriver::CreateServerWorkerEntity()
+{
+	const WorkerRequirementSet WorkerIdPermission{ { FString::Format(TEXT("workerId:{0}"), { Connection->GetWorkerId() }) } };
+
+	WriteAclMap ComponentWriteAcl;
+	ComponentWriteAcl.Add(SpatialConstants::POSITION_COMPONENT_ID, WorkerIdPermission);
+	ComponentWriteAcl.Add(SpatialConstants::METADATA_COMPONENT_ID, WorkerIdPermission);
+	ComponentWriteAcl.Add(SpatialConstants::ENTITY_ACL_COMPONENT_ID, WorkerIdPermission);
+	ComponentWriteAcl.Add(SpatialConstants::INTEREST_COMPONENT_ID, WorkerIdPermission);
+
+	QueryConstraint constraint;
+	constraint.EntityIdConstraint = SpatialConstants::INITIAL_GLOBAL_STATE_MANAGER_ENTITY_ID;
+
+	Query query;
+	query.Constraint = constraint;
+	query.FullSnapshotResult = true;
+
+	ComponentInterest queries;
+	queries.Queries.Add(query);
+
+	Interest interest;
+	interest.ComponentInterestMap.Add(SpatialConstants::POSITION_COMPONENT_ID, queries);
+
+	TArray<Worker_ComponentData> Components;
+	Components.Add(Position().CreatePositionData());
```

Comment:
@danielimprobable I'm a bit concerned about the side effects of this one and chunk-based interest. Is there something which says the entity should not be considered for CBI if it has the `Interest` component?

Reply:
Looks like entities with the `Interest` component are not considered in CBI so this shouldn't be an issue. In the future we might want to place the worker entities in a way which represents the zone each worker is responsible for to make it easier to visualise LB in the inspector.